ui.config/src/main/content/jcr_root/apps/groovyconsole-config/osgiconfig/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-groovyconsole.config

```diff
@@ -0,0 +1,13 @@
+scripts=[
+"
+create path /conf/groovyconsole/replication(sling:Folder)
+create path /conf/groovyconsole/scripts(sling:Folder)
+
+create path /var/groovyconsole/audit(nt:unstructured)
+
+create service user aem-groovy-console-service with path system/aem-groovy-console
+set ACL for aem-groovy-console-service
+    allow jcr:all on /
+    allow jcr:all on /var/groovyconsole/audit
```

Comment:
is there a deny that removes the previous jcr:all?

Reply:
no but there could be.. if a deny is put on /var for everyone we would like to keep this working at least (this is how it initially was..)